filebeat/prospector/runnerfactory.go

```diff
@@ -28,7 +27,6 @@ func NewRunnerFactory(outlet channel.Factory, registrar *registrar.Registrar, be
 func (r *RunnerFactory) Create(c *common.Config) (cfgfile.Runner, error) {
 	p, err := New(c, r.outlet, r.beatDone, r.registrar.GetStates())
 	if err != nil {
-		logp.Err("Error creating prospector: %s", err)
```

Comment:
LGTM. The only problem with the error logged by the runner factory is probably that it's more generic and does not talk about the prospector itself?

Reply:
One option would be to use `errors.Wrap` in `prospector.New`https://github.com/vjsamuel/beats/blob/e3e3770af3dd7d3f4563955274aff976f415d9cd/filebeat/prospector/prospector.go#L34

Reply:
Merged it as is as I kind of like producing errors in as few places and just pass them on. We can still improve it in case it becomes an issue.